package.json

```diff
@@ -284,6 +288,20 @@
               "args": ""
             }
           },
+          {
+            "label": "Java: Launch Program with Args Prompt",
+            "description": "Add a new configuration for launching a java program, prompting first for program arguments.",
+            "body": {
+              "type": "java",
+              "name": "Debug (Launch) with Args Prompt",
```

Comment:
Same with above, should not use abbreviation in the description to the end user.

Reply:
Debug (Launch) by Prompting Arguments